spyder/widgets/status.py

```diff
@@ -284,6 +284,28 @@ def update_interpreter(self, interpreter):
         self.update_tooltip()
 
 
+class ClockStatus(BaseTimerStatus):
+    """"Add clock to statusbar in a fullscreen mode"""
```

Comment:
Please add a period. Docstrings are sentences:

`""""Add clock to statusbar in a fullscreen mode."""`